Java/Data Type & Operators/1. Hello World/palak.java

```diff
@@ -0,0 +1,9 @@
+import java.util.*;
+
+public class palak
```

Comment:
Please rename and use capital letters for Class and File names, keep this consistant